translations/translate_survey.py

```diff
@@ -25,7 +25,7 @@ def dumb_to_smart_quotes(string):
 
     # Find dumb single quotes coming directly after letters or punctuation,
     # and replace them with right single quotes.
-    string = re.sub(r"([a-zA-Z0-9.,?!;:\"\'])'", r'\1’', string)
+    string = re.sub(r"([a-zA-ZÀ-ÿ0-9.,?!;:\"\'])'", r'\1’', string)
```

Comment:
What about:

> \w - When the LOCALE and UNICODE flags are not specified, matches any alphanumeric character and the underscore; this is equivalent to the set [a-zA-Z0-9_]. With LOCALE, it will match the set [0-9_] plus whatever characters are defined as alphanumeric for the current locale. If UNICODE is set, this will match the characters [0-9_] plus whatever is classified as alphanumeric in the Unicode character properties database.